src/geometry/manifold/complex.lean

```diff
@@ -0,0 +1,98 @@
+/-
+Copyright (c) 2022 Heather Macbeth. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Heather Macbeth
+-/
+import analysis.complex.abs_max
+import geometry.manifold.mfderiv
+
+/-! # Holomorphic functions on complex manifolds
+
+Thanks to the rigidity of complex-differentiability compared to real-differentiability, there are
+many results about complex manifolds with no analogue for manifolds a general normed field.  For
+now, this file contains just one such result:
+
+## Main results
+
+* `mdifferentiable.const_of_compact_space_of_connected_space`: A complex-differentiable function on
+  a compact preconnected complex manifold is constant.
+
+## TODO
+
+There is a whole theory to develop here, but an obvious next step would be to generalize the above
+result to possibly disconnected compact complex manifolds, where the result should be that a
+complex-differentiable function is `is_locally_constant`.  There are some gaps in the
+`is_locally_constant` API that need to be filled for this.
+-/
+
+open_locale manifold topological_space
+open complex
+
```

Comment:
How about:
```suggestion

namespace mdifferentiable

```
to save manually prepending `mdifferentiable.` in the lemma names.